Week2/homework/js-exercises/convertGrades.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+let gradeUsa = '';
+
+let score ='';
+//Created this if else function to compare score with different cases to find out USA equivelent of our score.
+function gradeCalculator(score) {
+    if (90 <= score && score <= 100) {
+    gradeUsa = 'A';
+    }
+        else if (80 <= score && score <= 89) {
+            gradeUsa = 'B';
+        }
+        else if (70 <= score && score <=79) {
+            gradeUsa = 'C';
+        }
+        else if (60 <= score && score <=69) {
+            gradeUsa = 'D';
+        }
+        else if (50 <= score && score <=59) {
+            gradeUsa = 'E';
+        }
+        else if (0 <= score && score <=49) {
+            gradeUsa = 'F';
+        }
+        
+
+
+
+return console.log('You got a '  + gradeUsa + ' (' + score + '%)!');;
```

Comment:
You don't need to return here, as `console.log` will always return `undefined`. It is enough just to do the console log and you can remove the `return` keyword.